rosdep/base.yaml

```diff
@@ -3395,6 +3395,7 @@ libuv-dev:
     xenial: [libuv0.10-dev]
     yakkety: [libuv0.10-dev]
     zesty: [libuv0.10-dev]
+    bionic: [libuv0.10-dev]
```

Comment:
As indicated by the CI failure and specified in the [contribution guide](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules) please move the entry for `bionic` to the top to preserve the alphabetic ordering.